db/1.8/modules/ROOT/pages/database-connector-connection.adoc

```diff
@@ -0,0 +1,393 @@
+  <db:oracle-connection
+  host="192.168.99.100"
+  port="1521" instance="xe"
+  user="system"
+  password="oracle" />
+</db:config>
+----
+
+== Configure the JDBC Driver
+
+After you configure the connections, you can add the JDBC driver. The following example shows how to add a driver for the MySQL database:
+
+. In Studio, you can add the driver using any of the following options:
+* In the *Database Config* window, click *Configure*, select *Add Maven dependency*, and pick the dependency:
++
+.Pick Maven dependency
+image::database-dependency.png[Pick Maven dependency]
++
+* Add the dependency directly in your project's `pom.xml` file:
++
+[source,xml,linenums]
+----
+<dependency>
+    <groupId>mysql</groupId>
+    <artifactId>mysql-connector-java</artifactId>
+    <version>5.1.48</version>
+</dependency>
+----
++
+. Through your `pom.xml` file, make Mule expose the driver dependency to the Database Connector:
++
+[source,xml,linenums]
+----
+<build>
+    <plugins>
+        <!-- Only used to declare the shared libraries-->
+        <plugin>
+            <groupId>org.mule.tools.maven</groupId>
+            <artifactId>mule-maven-plugin</artifactId>
+            <version>1.0.0</version>
+            <configuration>
+                <sharedLibraries>
+                    <sharedLibrary>
+                        <groupId>mysql</groupId>
+                        <artifactId>mysql-connector-java</artifactId>
+                    </sharedLibrary>
+                </sharedLibraries>
+            </configuration>
+        </plugin>
+    </plugins>
+</build>
+----
+
+== Configure Connection Pools
+
+Establishing connections is costly in terms of connectors, relational database management system (RDBMS) network, and processing overhead. Database Connector supports the *Pooling profile* parameter that enables you to build a connection pool to provide several connections.
+
+All connection type configurations, except the *Data Source Reference Connection*, accept the *Pooling profile* configuration. In JDBC, pooling occurs at the data source level. Therefore, to do pooling, you have to configure the global data source.
+
+The following example shows how to configure the *Pooling profile* parameter for a MySQL database connection:
+
+. In the *Database Config* window, in the *Advanced* tab, for *Pooling profile* select *Edit inline*.
+. Set *Max pool size* to `5`, *Min pool size* to `0`, *Acquire increment* to `1`, *Prepared statement cache size* to `5`, *Max wait* to `0`, *Max wait unit* to `SECONDS (Default)` and *Max idle time* to `0`.
+. Enable the *Additional properties* parameter. +
+These additional properties do not override any of the main properties, if any of those were previously set.
+. Click the (*+*) plus sign to add a new property.
+. In the *Additional property* window, set *Key* to `checkoutTimeout`, *Value* to `1000`, and click *Finish*.
+. Add a new property, set *Key* to `idleConnectionTestPeriod`, *Value* to `2`, and click *Finish*.
```

Comment:
```suggestion
. To add a new property, set *Key* to `idleConnectionTestPeriod` and *Value* to `2`, and then click *Finish*.
```
But...a second "Finish" step? Is this leftover/orig text that was not deleted when replaced?